portfolio.adobe.com.prod.json

```diff
@@ -0,0 +1,40 @@
+{
+  "providerId":"portfolio.adobe.com",
+  "providerName":"Adobe Portfolio",
+  "serviceId":"prod",
+  "serviceName":"Adobe Portfolio Websites",
```

Comment:
A very minor comment, nothing blocking, just to consider if the redundancy of "Adobe Portfolio" between `providerName` and `serviceId` is needed.

This is how it would look on IONOS consent screen:
![image](https://user-images.githubusercontent.com/28707030/66628599-93c72300-ebff-11e9-8cb0-94266da38f19.png)

Reply:
Thanks for pointing this out. We'll push an updated serviceName shortly.

Reply:
Updated. Thanks!